src/appleseed.studio/mainwindow/pythonconsole/pythoneditor.cpp

```diff
@@ -78,6 +78,8 @@ void PythonEditor::keyPressEvent(QKeyEvent* event)
         insert_spaces(4);
     else
     {
+        if (event->key() == Qt::Key_Return || event->key() == Qt::Key_Enter)
```

Comment:
I think shift+enter insert something like soft return in Microsoft Word (https://www.computerhope.com/jargon/s/softretu.htm) but I can't find anything in docs